_includes/commands/table-cmds.md

```diff
@@ -29,6 +30,7 @@
 **Examples**
 
 ```js
+tinymce.activeEditor.execCommand('mcetableapplycellstyle', false, { backgroundColor: 'red', borderColor: 'blue' });
```

Comment:
This command will need to be in camel case here as well.

I am wondering if we need some mention on how a style can be removed with an empty string and whether that should be another example like: `tinyMCE.activeEditor.execCommand('mceTableApplyCellStyle', false, { backgroundColor: '' })` or a sentence in the description further up the page.

Reply:
I think an example would be good here, yes, to show off what it can do

Reply:
Maybe where we say "applies" in the various sections, we could say "toggles" or "applies and removes"? Idk

Reply:
Recommended changes made in https://github.com/tinymce/tinymce-docs/pull/1570/commits/82e7d51f189765c65c7b9e80f3360d50a1dee960

Reply:
Looks good. Thanks Tyler.